canonical-web-and-design/vanilla-framework
# Clean up custom docs/site SCSS

## Done

- Move site and docs custom styles to one folder for easier building
- Remove unnecessary or unused styles
- Remove lazy loading from logo in main nav
- Improve watch scripts

Fixes #2899

## QA

- Run `./run` or demo
- Open http://0.0.0.0:8101/ or [demo](https://vanilla-framework-canonical-web-and-design-pr-2910.run.demo.haus/)
- Site should look and behave the same
- Run `./run watch` - all files (including standalone examples should be build on change)
- Run `./run exec yarn watch:scss:skip-standalone` only vanilla and website scss should be built
